content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -85,6 +85,9 @@ $ dvc stage add --name train --deps data/MNIST --deps train.py \
               --params seed,lr,weight_decay --live dvclive python train.py
 ```
 
+The `--live dvclive` option enables our special logger [DVCLive](/doc/dvclive),
```

Comment:
"special" is probably redundant ... there is nothing special about it :)

Reply:
It's special in that it registers checkpoints (it integrates with DVC).